### PR TITLE
Deploy FDB and stateless dhstore on `prod`

### PR DESCRIPTION
**Pull request by masih**
Set up FDB cluster and dhstore on prod. Use 5 storage servers with increased rate keeper limits tuned for write-heavy workload.

### CODE REVIEW

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/fdb-cluster/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      foundationdb.org/fdb-cluster-name: fdb-prod-cluster
+    processClassLabels:
+      - foundationdb.org/fdb-process-class
+    processGroupIDLabels:
+      - foundationdb.org/fdb-process-group-id
+  minimumUptimeSecondsForBounce: 60
+  processes:
+    general:
+      customParameters:
+        - memory=20GiB
```
Comment (gammazero) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/fdb-cluster/cluster.yaml:31]: Should this parameter be tuned to take up most or all of memory allocated to container?

**Review by gammazero — APPROVED** [review]
Would it be better to have a separate dhstore-stateless per indexer?

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/fdb-cluster/cluster.yaml (hunk 1 above)
Comment (masih) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/fdb-cluster/cluster.yaml:31]: `storageServersPerPod` are set to 3 which leaves 20 GiB ish per process. Right?

**Comment by masih** [comment]
> Would it be better to have a separate dhstore-stateless per indexer?

I don't think so. there is simply no need for it. Because we can independently scale dhstores this way since they have become stateless. This also means better utilsiation: if there is not much writes then we can scale down dhstores. 

Whereas if they are run one per indexer we cannot do this.

**Pull request merged**